docs/content/docs/connectors/datastream/kinesis.md

```diff
@@ -29,9 +29,26 @@ under the License.
 
 The Kinesis connector provides access to [Amazon AWS Kinesis Streams](http://aws.amazon.com/kinesis/streams/).
 
-To use the connector, add the following Maven dependency to your project:
-
-{{< artifact flink-connector-kinesis >}}
+To use this connector, add one or more of the following dependencies to your project, depending on whether you are reading from and/or writing to Kinesis Data Streams:
+
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left">KDS Connectivity</th>
+      <th class="text-left">Maven Dependency</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+        <td>Source</td>
+        <td>{{< artifact flink-connector-kinesis >}}</td>
+    </tr>
+    <tr>
+        <td>Sink</td>
+        <td>{{< artifact flink-connector-aws-kinesis-data-streams >}}</td>
+    </tr>
+  </tbody>
+</table>
 
 {{< hint warning >}}
 **Attention** Prior to Flink version 1.10.0 the `flink-connector-kinesis` has a dependency on code licensed under the [Amazon Software License](https://aws.amazon.com/asl/).
```

Comment:
It this still relevant for Flink 1.15? It's been a while since that change happened.

Reply:
You're right, is it time to remove it?

Reply:
I'd remove it. The comment only applies to Flink releases prior to 1.12 (I think).